File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if len(nums) < 3:
            return 2
        w = 2
        for i in range(2, len(nums)):
            if nums[w - 2] != nums[i]:
                nums[w] = nums[i]
                w += 1
        return w

File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.c
int removeDuplicates(int* nums, int numsSize) {
    if (numsSize <= 2)
        return numsSize;
    int r = 2;
    for (int i = 2; i < numsSize; ++i) {
        if (nums[i] != nums[r - 2]) {
            nums[r] = nums[i];
            r += 1;
        }
    }
    return r;
}